index.html

```diff
@@ -4,7 +4,76 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
+    <link rel="stylesheet" href="css/style.css" />
+
     <title>Portfolio</title>
   </head>
-  <body></body>
+  <body>
+    <header>
+      <div class="logo">
+        <a href="#">
+          <h1>My Logo</h1>
+        </a>
+      </div>
+      <div>
+        <nav>
+          <span class="bars">
+            <i class="fa-solid fa-bars"></i>
+          </span>
+        </nav>
+      </div>
+    </header>
+    <main>
+      <div class="hero">
+        <div>
+          <h1 class="hero-name">I'm Mariane</h1>
+        </div>
+        <div>
+          <h2 class="hero-msg">Glad to see you!</h2>
+        </div>
+        <div class="hero-minor-text">
+          <p>
+            I'm a software! I can help you build a product, feature or website
+            Look through some of my work and experience! If you like what you
+            see and have a project you need coded, don't hestiate to contact me.
+          </p>
+        </div>
+      </div>
+      <div>
+        <a href="#" class="text-connect">
+          <h5>LET'S CONNECT</h5>
+        </a>
```

Comment:
Just like the previous reviewer said, kindly use the button tag or a tag for this element since the purpose of the element is to connect to some site setup. 👍.

I think you can just remove the `h5 `tag and have just the` a ` tag
```
  <a href="#" class="text-connect">
         LET'S CONNECT
        </a>
```

Reply:
done :heavy_check_mark: